torch/_subclasses/fake_tensor.py

```diff
@@ -61,6 +61,14 @@
 from ._fake_tensor_utils import _CacheKeyState, _PySymInputStub, _SymIntOutputStub
 
 
+# Import fake_collectives to register meta implementations for c10d ops
+# This ensures operations like scatter_, broadcast_, etc. work in fake tensor mode
+try:
+    import torch.distributed._tools.fake_collectives  # noqa: F401
```

Comment:
oh you know what... we should be importing these meta registrations at `import torch` time, that way anybody that wants to run distributed ops with the meta device is able to do it. I had a PR here to do it that languished: https://github.com/pytorch/pytorch/pull/162119. If you want to take it over and/or move the code into this PR feel free!